src/core/lib/iomgr/tcp_server_posix.cc

```diff
@@ -203,6 +203,7 @@ static void on_read(void* arg, grpc_error_handle err) {
     if (fd < 0) {
       switch (errno) {
         case EINTR:
+        case ECONNABORTED:
```

Comment:
This is a good change. However, I'd think we should treat this just like `EAGAIN`.

There is also `EWOULDBLOCK ` which we probably should handle the same way.

Reply:
Done!